GroceryStore/GroceryStore.Models/Customer.cs

```diff
@@ -0,0 +1,113 @@
+﻿using GroceryStore.Constants;
+using System.Diagnostics;
+using System.Dynamic;
+using System.Globalization;
+using System.Transactions;
+using System.Xml.Linq;
+
+namespace GroceryStore.Models
+{
+    public class Customer
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int Age { get; set; }
+        public string Sex { get; set; }
+
+        private bool _hasDiscountCard;
+        public bool HasDiscountCard
+        {
+            get
+            { return _hasDiscountCard; }
+            set
+            { _hasDiscountCard = value; }
+        }
+        public string DiscountCardToString
+        {
+            get
+            {
+                return _hasDiscountCard ? "Yes" : "No";
+            }
+        }
+        private double _personalDiscount;
+        public double PersonalDiscount
+        {
+            get
+            {
+                return (HasDiscountCard ? _personalDiscount : 0);
+            }
+            set
+            {
+                _personalDiscount = value;
+            }
+        }
+        public string PercentageAsString
+        {
+            get
+            {
+                return (PersonalDiscount * 100).ToString("0.##") + "%";
+            }
+        }
+        public string FullName
+        {
+            get
+            {
+                return $"{FirstName} {LastName}";
+            }
+        }
+        public Product[] Cart { get; set; }
+        public int CartCount { get; set; }
+
+
+        public Customer(string firstName, string lastName, int age, string sex, bool hasDiscountCard, double personalDiscount)
+        {
+            FirstName = firstName;
+            LastName = lastName;
+            Age = age;
+            Sex = sex;
+            HasDiscountCard = hasDiscountCard;
+            PersonalDiscount = personalDiscount;
+            Cart = new Product[10];
+            CartCount = 0;
+        }
```

Comment:
A little bit about clean code :smirk:
Methods must be separated by a blank line.